Source/BridgeDelegate.swift

```diff
@@ -117,7 +117,7 @@ public final class BridgeDelegate: BridgingDelegate {
     @discardableResult
     public func bridgeDidReceiveMessage(_ message: Message) -> Bool {
         guard destinationIsActive,
-              location == message.metadata?.url else {
+              webView?.url?.absoluteString ?? location == message.metadata?.url else {
             logger.warning("bridgeDidIgnoreMessage: \(String(describing: message))")
```

Comment:
Nice solution. 
I've left some feedback in https://github.com/hotwired/strada-ios/pull/25.